app/scripts/views/marketing_snippet_ios.js

```diff
@@ -85,22 +85,26 @@ define([
       MarketingSnippetView.prototype.initialize.call(this, options);
 
       this._language = options.language;
-      this._able = options.able;
+
+      // ** Will be needed when Fx iOS launches
+      //this._able = options.able;
     },
 
     context: function () {
+      // ** Commented content below will be needed when Fx iOS launches
+
       // fallback in case experiment has not concluded
-      var fxIos = this._able.choose('fxIos') || {};
+      //var fxIos = this._able.choose('fxIos') || {};
```

Comment:
Out of curiosity, does the commented code "matter", or won't it be stripped from the minified source anyways?